wdfn-server/waterdata/templates/monitoring_location.html

```diff
@@ -78,48 +78,70 @@ <h1>{{ stations[0].station_nm }}</h1>
                     <p id="site-description">{{ components.Description(stations[0].site_no, location_with_values, parm_grp_summary) }}</p>
                 </div>
 
-                {{ components.TimeSeriesComponent(stations[0].site_no, stations[0].dec_lat_va, stations[0].dec_long_va) }}
+                {% if request.user_agent.browser == 'msie' %}
+                    {% block javascript %}
+                        <script>
+                            var urlString = window.location.href;
```

Comment:
I am a little worried about putting javascript here that selects a node. It might be better to wrap this in a document.addEventListener('DOMContentLoaded', function(event) {......})

Reply:
Makes sense--will do.